site/content/docs/5.1/examples/blog/index.html

```diff
@@ -207,7 +207,7 @@ <h2 class="blog-post-title">New feature</h2>
 
       <nav class="blog-pagination" aria-label="Pagination">
         <a class="btn btn-primary" href="#">Older</a>
-        <a class="btn btn-secondary disabled" href="#" tabindex="-1" aria-disabled="true">Newer</a>
+        <a class="btn btn-outline-secondary disabled">Newer</a>
```

Comment:
![image](https://user-images.githubusercontent.com/11272238/133741720-6e978f8b-f4ac-4978-95f3-c6a32b74bcd4.png)
there's no `.btn-outline` style in boosted. Could keep it at time and update it when reviewing samples